src/gui/networksettings.cpp

```diff
@@ -164,6 +164,9 @@ void NetworkSettings::saveProxySettings()
         cfgFile.setProxyType(QNetworkProxy::DefaultProxy);
     } else if (_ui->manualProxyRadioButton->isChecked()) {
         int type = _ui->typeComboBox->itemData(_ui->typeComboBox->currentIndex()).toInt();
+        QString host = _ui->hostLineEdit->text();
+        if (host.isEmpty())
```

Comment:
Didn't @SamuAlfageme somewhere mention to .trim() the hostname too?

Reply:
Hm, I didn't see that.